apps/next/app/server/db/schemaForEnv.ts

```diff
@@ -3,6 +3,8 @@ import os from "os";
 export default function sqlSchemaForEnv(env: string) {
   const username = os.userInfo().username;
   switch (env) {
+    case "ci":
+      return `metaldev_${username}`;
```

Comment:
I don't think this is what we want? The OS username in CI could be anything? I think this function should probably take in the CI env variable as an additional argument, and if it is 'true', then set username to something made up like "githubaction". Then I think all the rest can remain the same

Reply:
Well no we actually don't need the "ci" case anymore.

Reply:
That said, I'll use the CI variable to choose a made-up username as advised.

Reply:
Done.